src/charts/line.js

```diff
@@ -463,6 +467,8 @@ define(function(require){
               .append('g').classed('axis y', true);
             container
               .append('g').classed('grid-lines-group', true);
+            container
+                .append('g').classed('custom-lines-group', true);
```

Comment:
I am doubting if we should think this as a 'metadata' element.
In the future, it could be part of an 'annotations' group.

This is good for now though.